luigi/contrib/pyspark_runner.py

```diff
@@ -36,11 +36,66 @@
 import sys
 import os
 
+from luigi import configuration
+
 # this prevents the modules in the directory of this script from shadowing global packages
 sys.path.append(sys.path.pop(0))
 
 
-class PySparkRunner(object):
+class SparkEntryPoint(object):
```

Comment:
Can you make this private? (rename to `_SparkEntryPoint`?

Reply:
thanks for your response, I think this makes sense